packages/vanilla/src/sass/utilities/_spacing.scss

```diff
@@ -4,20 +4,29 @@
 
 $spacing: (
   16x: var(--cbp-space-16x),
+  15x: var(--cbp-space-15x),
+  14x: var(--cbp-space-14x),
+  13x: var(--cbp-space-13x),
   12x: var(--cbp-space-12x),
+  11x: var(--cbp-space-11x),
+  10x: var(--cbp-space-10x),
+  9x: var(--cbp-space-9x),
   8x: var(--cbp-space-8x),
+  7x: var(--cbp-space-7x),
   6x: var(--cbp-space-6x),
   5x: var(--cbp-space-5x),
   4x: var(--cbp-space-4x),
   3x: var(--cbp-space-3x),
   2x: var(--cbp-space-2x),
   1x: var(--cbp-space-1x),
+  half-x: var(--cbp-space-half-x),
+  1px: 1px,
   0: 0
 );
 
 @each $name, $space in $spacing {
   .cbp-margin-#{$name} {
-    margin-left: #{$space} !important;
+    margin: #{$space} !important;
```

Comment:
Why do these use !important? This is an anti-pattern and breaks CSS specificity. These are utility classes and should act as such (classes).